lib/matplotlib/quiver.py

```diff
@@ -179,7 +179,7 @@
     Minimum length as a multiple of shaft width; if an arrow length
     is less than this, plot a dot (hexagon) of this diameter instead.
 
-color : color or color sequence, optional
+color : :mpltype:`color` or color sequence, optional
```

Comment:
Should `color sequence` here be `list of :mpltype:color`?

Reply:
On a general note, I've used a couple of search/replace patterns. It is likely that I have not found all color type definitions with that. I think it's enough to start with the bulk here and fix others later if we find them.